02_cancer_type_classification/lasso_range_analysis/nbconverted/lasso_range_all.py

```diff
@@ -204,40 +204,52 @@ def get_top_and_smallest_diff(gene, cancer_type):
              'smallest_lasso_param', 'top_smallest_diff']
 )
 
+all_top_smallest_diff_df['best'] = 'top'
+all_top_smallest_diff_df.loc[
+    all_top_smallest_diff_df.top_smallest_diff < 0, 'best'
+] = 'smallest'
+all_top_smallest_diff_df.loc[
+    all_top_smallest_diff_df.top_smallest_diff == 0, 'best'
+] = 'zero'
+
+print(all_top_smallest_diff_df.best.value_counts())
 all_top_smallest_diff_df.head()
 
 
-# In[9]:
+# In[10]:
 
 
 sns.set({'figure.figsize': (8, 6)})
+sns.set_style('whitegrid')
 
 sns.histplot(all_top_smallest_diff_df.top_smallest_diff)
-plt.title('Differences between top and smallest LASSO parameter')
-plt.xlabel('top - smallest')
+plt.title('Differences between "best" and "smallest good" LASSO parameter')
```

Comment:
Do you define anywhere what "smallest good" is?

Reply:
Nope, not in the notebooks - I'll add some documentation.